components/closebutton/metadata/closebutton.yml

```diff
@@ -1,25 +1,19 @@
 name: Close button
 status: Verified
 description: |
-  A button used to close or dismiss components.
+  A button used to close or dismiss components. Close button supports different sized icons using these class couplings:
+
+  | Component size  | Close button classname          | UI icon classname           |
+  | --------------  | ------------------------------- | --------------------------- |
+  | Small           | `.spectrum-CloseButton--sizeS`  | `.spectrum-UIIcon-Cross400` |
+  | Medium (default)| No size class for default       | `.spectrum-UIIcon-Cross500` |
+  | Large           | `.spectrum-CloseButton--sizeL`  | `.spectrum-UIIcon-Cross400` |
+  | X-Large         | `.spectrum-CloseButton--sizeXL` | `.spectrum-UIIcon-Cross500` |
+
 sections:
   - name: Custom Properties API
     description: |
       This component can be modified via its `--mod-*` prefixed custom properties. A list of those prefixed custom properties can be found <a class="spectrum-Link" href="https://github.com/adobe/spectrum-css/tree/main/components/closebutton/metadata/mods.md">here</a>.
-  - name: Migration Guide
-    description: |
-      ### Sizing
-      Close button supports different sized icons. By default, you should use the following icons:
-
-      | Close button classname          | UI icon classname           |
-      | ------------------------------- | --------------------------- |
-      | `.spectrum-CloseButton--sizeS`  | `.spectrum-UIIcon-Cross75`  |
-      | `.spectrum-CloseButton--sizeM`  | `.spectrum-UIIcon-Cross100` |
-      | `.spectrum-CloseButton--sizeL`  | `.spectrum-UIIcon-Cross200` |
-      | `.spectrum-CloseButton--sizeXL` | `.spectrum-UIIcon-Cross300` |
-
-      ### Remove focus-ring class
-      We've migrated away from the focus-ring class in favor of the native `:focus-visible` pseudo-class due to changes in browser support.
 examples:
   - id: close
     name: Icon Size - Regular
```

Comment:
Since we're removing the "Icon Size - Large" heading, I'm wondering if there are guidelines for changing this text as well? Looking at other components, it seems like "Sizing" could be a more appropriate heading here.